lib/elixir/lib/list.ex

```diff
@@ -577,6 +577,24 @@ defmodule List do
   end
 
   @doc """
+  Returns true if `list` starts with `prefix`.
```

Comment:
`prefix` is better because it is not a single element but a list. My suggestion:

> Returns true if `list` starts with the given `prefix` list.